frontend/components/icons/ExternalLink.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 the JSR authors. All rights reserved. MIT license.
+export function ExternalLink(props: { class?: string }) {
+    return (
+        <svg
+          fill="none"
+          stroke-width="1.5"
```

Comment:
I think it'd be nice to have a stroke width prop on this so it could be increased to 2.25 when it sits next to bold text like it does in this pill. 1.5 is a good default but would be nice to override that in this implementation

<img width="323" alt="Screenshot 2024-07-05 at 10 10 50 AM" src="https://github.com/jsr-io/jsr/assets/776987/21d158b6-b98a-45af-a2e5-8ee264953e22">

Reply:
You got it. I've made these changes, let me know if you need anything else. :)